main/utils/accelerator-validator.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
```

Comment:
Well there are some specific rules about what accelerators are valid and I wanted them to be handled. Also the `event.which` and `event.code` work really weirdly with some meta keys (like option or shift). Also, with the current implementation you weren't able to register num keys, as we used the `event.which` which would register for example `Cmd+5` instead of `Cmd+num5`

For all these edge cases, I decided to make a full validation utility based on electron's docs

Reply:
I would write a couple of short sentences about it as a code comment on the top of this file. Just for future readers.